lib/omnibus/version.rb

```diff
@@ -15,5 +15,5 @@
 #
 
 module Omnibus
-  VERSION = '4.0.0'
+  VERSION = '4.0.1'
```

Comment:
Please do not bump versions like this in PRs. That is a release time task of which this change might be batched up with others.